cli/ballerina-cli/src/main/resources/cli-help/ballerina-graphql.help

```diff
@@ -35,6 +41,14 @@ OPTIONS
             GraphQL schema. This option is used with the GraphQL schema generation command.
             If this base path is not specified, schemas will be generated for each of the GraphQL
             services in the input file.
+        -m, --mode
+            This mode is used to identify the operation mode. It can be `client`, `schema`, or `service`. The `client` argument
+            indicates the Ballerina client source code generation, the `schema` argument indicates the GraphQL schema
+            generation, and the `service` argument indicates the Ballerina GraphQL service source code generation. Mode is only 
+            mandatory for Ballerina GraphQL service source code generation.
+        -r, --use-records-for-objects
+            This flag is used without an argument. It is used only in the Ballerina GraphQL service generation. It will make 
+            the Ballerina CLI tool to use record types for GraphQL object types whenever possible.
```

Comment:
Let's follow the character limit on the above sections here.

Reply:
@DimuthuMadushan maybe we can fork @chathuranga-jayanath-99's repo and do the changes on top of that. Then send a new PR, if changing this is not possible.

Reply:
I'll add the changes here.